.github/workflows/bmvn_build_appcenter_android.yaml

```diff
@@ -39,6 +32,42 @@ jobs:
   build:
     runs-on: [ self-hosted, macm2, build-ci-local-bmvn, android ]
     steps:
+      - name: Extract Build number for Testnet
+        id: extract-version-testnet
+        env:
+            APPCENTER_API_TOKEN: ${{ secrets.APPCENTER_API_TOKEN }}
+        run: |
+            # Make the API request and extract the short_version using jq
+            API_RESPONSE=$(curl -X GET \
+            "https://api.appcenter.ms/v0.1/apps/support-zzd0-28/Autonomy-Android-Test/recent_releases" \
```

Comment:
can we combine 2 steps into 1 by checking testnet condition to replace the app name `Autonomy-Android-Test` or `Autonomy-Android-1`?

Reply:
I combined but it has problem in calling correct URL to Appcenter. So I changed to this style. In the past, it was very shortly.

Reply:
does this work?

```
env:
          APPCENTER_API_TOKEN: ${{ secrets.APPCENTER_API_TOKEN }}
          APP_NAME: ${{ inputs.testnet == true ? 'Autonomy-Android-Test' : 'Autonomy-Android-1' }}
        run: |
          API_RESPONSE=$(curl -X GET \
          "https://api.appcenter.ms/v0.1/apps/support-zzd0-28/$APP_NAME/recent_releases" \
          -H 'accept: application/json' \
          -H "X-API-Token: $APPCENTER_API_TOKEN")

          echo "$API_RESPONSE"
          BUILD_NUMBER=$(echo "$API_RESPONSE" | jq -r '.[0].version')
          # Increase the build number by 1
          ((BUILD_NUMBER++))
          echo "$BUILD_NUMBER"
          echo "FLUTTER_VERSION_CODE=$BUILD_NUMBER" >> "$GITHUB_ENV"
```

Reply:
I changed the way to check.